test/fixtures/rules/std_rule_cases/L065.yml

```diff
@@ -0,0 +1,107 @@
+rule: L065
+
+test_fail_simple_fix_union_all_before:
+  fail_str: |
+      SELECT 'a' UNION ALL
+      SELECT 'b'
+  fix_str: |
+      SELECT 'a'
+      UNION ALL
+      SELECT 'b'
+
+test_fail_simple_fix_union_all_after:
+  fail_str: |
+      SELECT 'a'
+      UNION ALL SELECT 'b'
+  fix_str: |
+      SELECT 'a'
+      UNION ALL
+      SELECT 'b'
+
+test_fail_simple_fix_union_all_before_and_after:
+  fail_str: |
+      SELECT 'a' UNION ALL SELECT 'b'
+  fix_str: |
+      SELECT 'a'
+      UNION ALL
+      SELECT 'b'
+
+test_pass_multiple_newlines_are_allowed:
+  pass_str: |
+      SELECT 'a'
+
+
+      UNION ALL
+
+
+      SELECT 'b'
+
+test_fail_fix_works_in_subqueries:
+  fail_str: |
+      SELECT * FROM (
+          SELECT 'g' UNION ALL
+          SELECT 'h'
+          UNION ALL SELECT 'j'
+      )
+  fix_str: |
+      SELECT * FROM (
+          SELECT 'g'
+      UNION ALL
+          SELECT 'h'
+          UNION ALL
+      SELECT 'j'
+      )
```

Comment:
This is extremely ugly. I note your TODO above, and maybe it's fine if the other rule fixes it, but think we should do a few extra steps here:
1. Note that the result is ugly but will be fixed by other rule.
2. I presume you've tested and it IS fixed by other rule?
3. Maybe add a test as well to `test/rules` to show it fixes it properly?
4. Add another "pass" test with subquery (with correct indentation) to show it passes.

Reply:
Done, L003 indeed fixes the SQL as expected. Let me know what you think.